Standards/scs-0112-v1-volume-type-standard.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Volume Type Standard
+type: Standard
+status: Draft
+track:  IaaS
+---
+
+## Introduction
+
+Volume Types are used to classify volumes and provide a basic decision for what kind of volume should be created. These volume types can sometimes be very backend-specific and it might be hard for a user to choose the most suitable volume type, if there is more than one default type.
+
+### Glossary
+
+The following special terms are used throughout this standard document:
+| Term | Meaning |
+|---|---|
+| volume | OpenStack ressource, virtual drive which usually resides in a network storage backend |
+| AZ | Availability Zone |
+| Volume QoS | Quality of Service object for Volumes |
+
+## Motivation
+
+We want to standardize a few varieties of volume types. While a user can choose simple things like size when creating a volume, Volume Types define a few broader aspects of a volume. Encryption of volumes for example is solely decided by the volume type. While the option with which the volume will be replicated is a mix between definition in the volume type and backend specific configuration, but it's visibility can only be reached in the volume type. In the following part, we want to state, which varieties of volume types are REQUIRED, RECOMMENDED or only OPTIONAL within an SCS-compatible deployment.
+
+## Design Considerations
+
+All Considerations can be looked up in detail in the [Decision Record for the Volume Type Standard.](https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0111-v1-volume-type-decisions.md)
+
+To test whether a deployment has volume types with certain aspects, the discoverability of the parameters in the volume type has to be given. The following table shows, which aspects are considered in this standard. Furthermore the table shows, according to the current upstream implementation, whether the feature is discoverable, and who can discover this. The comments state what can be done to help with the discoverability and other important restrictions. The last column says how the standard is handled, which is currently through the addition of a standardized phrase at the beginning of the description of a volume type.
+
+| Aspect | Part of Standard | Discoverability | comments | concluded standardized description |
```

Comment:
Content of this table needs refinement:

- What does "Upstream work for ..." mean? Does this means, that upstream is currently working on this feature **or**   that this feature has to be implemented by someone else. Does "Upstream: setable (Typo?) ..." mean the same? 
- "backend-dependent" is too generic and does not answer the question of discoverability. A reference to more details would be helpful.
-  Discoverability: Does "yes" mean, that everybody can discover this property?

Just a minor request for change in favor of visibility: Either use lower-case upper case in column "Aspect" don't mix both.

Reply:
The table changed within the last commits.